airbyte-integrations/connectors/source-iterable/unit_tests/test_source.py

```diff
@@ -2,27 +2,17 @@
 # Copyright (c) 2023 Airbyte, Inc., all rights reserved.
 #
 
-from unittest.mock import MagicMock, patch
-
 import pytest
 import responses
 from source_iterable.source import SourceIterable
-from source_iterable.streams import Lists
 
 
 @responses.activate
-@pytest.mark.parametrize("body, status, expected_streams", ((b"", 401, 7), (b"alpha@gmail.com\nbeta@gmail.com", 200, 44)))
-def test_source_streams(mock_lists_resp, config, body, status, expected_streams):
+@pytest.mark.parametrize("body, status, expected_streams", [(b"alpha@gmail.com\nbeta@gmail.com", 200, 44)])
+def test_source_streams(config, body, status, expected_streams):
+    responses.get("https://api.iterable.com/api/lists", json={"lists": [{"id": 1}]})
     responses.add(responses.GET, "https://api.iterable.com/api/lists/getUsers?listId=1", body=body, status=status)
-    streams = SourceIterable().streams(config=config)
-    assert len(streams) == expected_streams
-
-
-def test_source_check_connection_ok(config):
```

Comment:
Can we patch at the http request level instead of deleting the `test_source_check_connection_*` tests?

I know that for some streams we're using the low code component, but since it looks like we're not migrating all streams it feels like a good idea to keep the coverage.

Reply:
Added